nix/supervisord-cluster/default.nix

```diff
@@ -134,20 +134,41 @@ let
     ++ optionals (!workbenchDevMode)
     [ workbench.workbench ];
 
+  startClusterUsage = ''
+Usage:
+   start-cluster [FLAGS..]
+
+   Flags:
+
+      --batch-name NAME               Override the batch name (default: ${batchName})
+      --no-generator | --no-gen       Don't auto-start the tx-generator
+
+      --trace | --debug               Trace the start-cluster script
```

Comment:
is there is difference between `--trace` and `--debug` ?

Reply:
No, none.

Reply:
This confusing. There should not be two different flags that actually do the same thing.
Please choose one of the two.